modules/account-lib/package.json

```diff
@@ -31,6 +31,7 @@
     "@bitgo/sdk-coin-atom": "^3.6.11",
     "@bitgo/sdk-coin-avaxc": "^2.7.0",
     "@bitgo/sdk-coin-avaxp": "^3.7.11",
+    "@bitgo/sdk-coin-bera": "^1.0.0",
```

Comment:
Unless there is a specific use case for using this in account-lib elsewhere, let's remove it.

Reply:
It's needed for wallet-platform, same for the other coins being added